pkg/api/transaction.go

```diff
@@ -183,8 +183,8 @@ func (s *Service) transactionCancelHandler(w http.ResponseWriter, r *http.Reques
 
 	txHash, err := s.transaction.CancelTransaction(ctx, paths.Hash)
 	if err != nil {
-		logger.Debug("cancel transaction failed", "tx_hash", txHash, "error", err)
```

Comment:
maybe also include the original and the new cancellation transaction in the logs